src/engraving/rendering/dev/tlayout.cpp

```diff
@@ -1531,14 +1531,14 @@ void TLayout::layoutBracket(const Bracket* item, Bracket::LayoutData* ldata, con
         double x = -w;
 
         double bd = (conf.styleSt(Sid::MusicalSymbolFont) == "Leland") ? spatium * .5 : spatium * .25;
-        ldata->shape.add(RectF(x, -bd, w * 2, 2 * (ldata->bracketHeight * 0.5 + bd)));
-        ldata->shape.add(item->symBbox(SymId::bracketTop).translated(PointF(-w, -bd)));
-        ldata->shape.add(item->symBbox(SymId::bracketBottom).translated(PointF(-w, bd + ldata->bracketHeight)));
-
-        w += item->symWidth(SymId::bracketTop);
         double y = -item->symHeight(SymId::bracketTop) - bd;
         double h = (-y + ldata->bracketHeight * 0.5) * 2;
-        ldata->setBbox(RectF(x, y, w, h));
+        ldata->setBbox(RectF(x, y, w + item->symWidth(SymId::bracketTop), h));
```

Comment:
I think this call should be removed completely. Now that Shape and Bbox are unified, `setBbox` is unnecessary if you also call `setShape`, because `setBbox` basically means `setShape("a shape consisting of only the given rect")` (see its definition in engravingitem.cpp).